tools/slurm/scripts/slurmcluster.sh

```diff
@@ -76,7 +76,7 @@ while [[ $# -gt 0 ]]; do
             echo '  -A '
             echo "           Description: Invokes a slurmcluster that uses agents instead of the launcher."
             echo "           Example: $0 -A"
-            echo '  -c {enroot|podman|singularity}'
+            echo '  -c {docker}'
```

Comment:
I think ```enroot|podman|singularity``` are the correct options, since we don't support docker with launcher and we still will support the other runtimes for launcher